### PR TITLE
Conda Pipeline Adjustments

### PR DESCRIPTION
**Pull request by scbedd**
Responding to feedback. Thanks @mccoyp !

@xiangyan99 if you have any further questions, asks, etc, I will take action on them here.

### CODE REVIEW

**Comment by scbedd** [comment]
[Changes check out](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3092563&view=results)

File: tools/azure-sdk-tools/ci_tools/conda/conda_functions.py [hunk 1]
```diff
@@ -79,8 +79,6 @@
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
```
Comment (mccoyp) [inline tools/azure-sdk-tools/ci_tools/conda/conda_functions.py:83]: Just double-checking; is 3.6 also still supported by anyone?

**Review by mccoyp — APPROVED** [review]
Looks good! Just one question about version support.

File: tools/azure-sdk-tools/ci_tools/conda/conda_functions.py (hunk 1 above)
Comment (scbedd) [reply tools/azure-sdk-tools/ci_tools/conda/conda_functions.py:83]: This is a good question. **NO** is the answer. Update inc.

**Pull request merged**